test_conformance/printf/test_printf.cpp

```diff
@@ -844,11 +1018,25 @@ test_definition test_list[] = {
     ADD_TEST(float),         ADD_TEST(float_limits),  ADD_TEST(octal),
     ADD_TEST(unsigned),      ADD_TEST(hexadecimal),   ADD_TEST(char),
     ADD_TEST(string),        ADD_TEST(format_string), ADD_TEST(vector),
-    ADD_TEST(address_space), ADD_TEST(buffer_size),
+    ADD_TEST(address_space), ADD_TEST(buffer_size),   ADD_TEST(mixed_format),
```

Comment:
```suggestion
    ADD_TEST(address_space), ADD_TEST(buffer_size),   ADD_TEST(mixed_format_random),
```

To ensure the word "random" shows up in the user-visible test name too.

Reply:
done